src/dbally/_collection.py

```diff
@@ -226,3 +228,29 @@ async def ask(self, question: str, dry_run: bool = False, return_natural_respons
         await event_tracker.request_end(RequestEnd(result=result))
 
         return result
+
+    def get_similarity_indexes(self) -> Dict[AbstractSimilarityIndex, List[Tuple[str, str, str]]]:
+        """
+        List all similarity indexes from all views in the collection.
+
+        Returns:
+            Dictionary with similarity indexes as keys and values containing lists of places where they are used
+            (represented by a tupple containing view name, method name and argument name)
+        """
+        indexes: Dict[AbstractSimilarityIndex, List[Tuple[str, str, str]]] = {}
+        for view_name in self._views:
+            view = self.get(view_name)
+            filters = view.list_filters()
+            for filter_ in filters:
+                for param in filter_.parameters:
+                    if param.similarity_index:
+                        indexes.setdefault(param.similarity_index, []).append((view_name, filter_.name, param.name))
```

Comment:
nit: `defaultdict` from `collections` is helpful

Reply:
I know of defaultdict, but since here the object is part of our public interface I have a slight preference for returning the standard dictionary, to avoid any surprises from the users regarding a non-standard behavior from the returned object (and/or having to document the non-standard behavior).